tests/tools/onert_train/src/args.cc

```diff
@@ -222,20 +257,17 @@ void Args::Initialize(void)
     )
     ("mem_poll,m", po::value<bool>()->default_value(false)->notifier([&](const auto &v) { _mem_poll = v; }), "Check memory polling (default: false)")
     ("epoch", po::value<int>()->default_value(5)->notifier([&](const auto &v) { _epoch = v; }), "Epoch number (default: 5)")
-    ("batch_size", po::value<int>()->default_value(32)->notifier([&](const auto &v) { _batch_size = v; }), "Batch size (default: 32)")
-    ("learning_rate", po::value<float>()->default_value(0.001)->notifier([&](const auto &v) { _learning_rate = v; }), "Learning rate (default: 0.001)")
-    ("loss", po::value<int>()->default_value(1)->notifier([&] (const auto &v) { _loss_type = v; }),
-        "Loss type\n"
-        "1: MEAN_SQUARED_ERROR (default)\n"
-        "2: CATEGORICAL_CROSSENTROPY")
-    ("loss_reduction_type", po::value<int>()->default_value(1)->notifier([&] (const auto &v) { _loss_reduction_type = v; }),
-        "Loss Reduction type\n"
-        "1: SUM_OVER_BATCH_SIZE(default)\n"
-        "2: SUM")
-    ("optimizer", po::value<int>()->default_value(1)->notifier([&] (const auto &v) { _optimizer_type = v; }),
-      "Optimizer type\n"
-      "1: SGD (default)\n"
-      "2: Adam")
+    ("batch_size", po::value<int>()->notifier([&](const auto &v) { _batch_size = v; }), "Batch size (default: use model file parameter)\n")
```

Comment:
(suggestion)
How about setting a default value for batch_size? Since the batch size is not related to the model file, so onert_train can provide a default batch size value to the user.

TensorFlow sets 32 as the default batch size, and PyTorch sets 1 as the default batch size. In ONERT, if the data length is not divisible by the batch size, it does not train the rest of the dataset. Therefore, I suggest setting 1 as the batch size for onert_train.

Reply:
I updated like this : [`f0e231d` (#12445)](https://github.com/Samsung/ONE/pull/12445/commits/f0e231d30761c640dd7b096ffd173c1d27cfea45) 

Since the onert_train main function has to know whether the value of `batch_size` is user-given or not, It is hard to remove `optional` here. 
I updated batch size decision logic in the onert_train main function. 

Is this way fine with you?